lib/active_remote/primary_key.rb

```diff
@@ -48,7 +48,8 @@ def primary_key
     #   person = Person.new(1)
     #   person.to_key # => [1]
     def to_key
-      send(primary_key) ? [send(primary_key)] : nil
+      @__to_key_key = respond_to?(primary_key) && send(primary_key) if @__to_key_key.nil?
```

Comment:
Weird name to avoid collisions, and it seemed like if this returns `false` that we'd still want to cache that, hence the nil check as opposed to an `||=`.